spec/controllers/feedbacks_controller_spec.rb

```diff
@@ -0,0 +1,121 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe FeedbacksController do
+  include LoginMacros
+
+  describe "POST #create" do
+    let(:mock_zoho) { instance_double(ZohoResourceClient) }
+
+    let(:default_parameters) do
+      {
+        feedback: {
+          comment: "Hello",
+          email: "test@example.com",
+          summary: "Summary",
+          language: "en"
+        }
+      }
+    end
+
+    before do
+      allow_any_instance_of(Feedback).to receive(:zoho_enabled?).and_return(true)
+      allow(mock_zoho).to receive(:retrieve_contact_id)
+      allow_any_instance_of(FeedbackReporter).to receive(:zoho_resource_client).and_return(mock_zoho)
+    end
+
+    context "when accessed by a logged-in user" do
+      let(:user) { create(:user) }
+
+      before do
+        fake_login_known_user(user)
+      end
+
+      context "and the user has no skin set" do
```

Comment:
Can you swap the "and" in the context descriptions for "when"? It's awkward grammar if we're writing it out as a sentence, but it's consistent with what we do in other specs.